extensions/parquet/table/src/main/java/io/deephaven/parquet/table/pushdown/BytePushdownHandler.java

```diff
@@ -0,0 +1,148 @@
+//
+// Copyright (c) 2016-2025 Deephaven Data Labs and Patent Pending
+//
+// ****** AUTO-GENERATED CLASS - DO NOT EDIT MANUALLY
+// ****** Edit CharPushdownHandler and run "./gradlew replicateParquetPushdownHandlers" to regenerate
+//
+// @formatter:off
+package io.deephaven.parquet.table.pushdown;
+
+import io.deephaven.api.filter.Filter;
+import io.deephaven.engine.table.impl.select.ByteRangeFilter;
+import io.deephaven.engine.table.impl.select.MatchFilter;
+import io.deephaven.util.QueryConstants;
+import io.deephaven.util.annotations.InternalUseOnly;
+import io.deephaven.util.type.TypeUtils;
+import org.jetbrains.annotations.NotNull;
+
+import java.util.Arrays;
+
+@InternalUseOnly
+public abstract class BytePushdownHandler {
+
+    public static boolean maybeOverlaps(
+            final Filter filter,
+            final MinMax<?> minMax) {
+        final byte min = (Byte) minMax.min();
+        final byte max = (Byte) minMax.max();
+        if (filter instanceof ByteRangeFilter) {
+            return maybeOverlaps(min, max, (ByteRangeFilter) filter);
+        } else if (filter instanceof MatchFilter) {
+            return maybeMatches(min, max, (MatchFilter) filter);
+        }
+        return true;
+    }
+
+    static boolean maybeOverlaps(
+            final byte min,
+            final byte max,
+            final ByteRangeFilter byteRangeFilter) {
+        // Skip pushdown-based filtering for nulls
+        final byte dhLower = byteRangeFilter.getLower();
+        final byte dhUpper = byteRangeFilter.getUpper();
+        if (dhLower == QueryConstants.NULL_BYTE || dhUpper == QueryConstants.NULL_BYTE) {
+            return true;
+        }
```

Comment:
Shouldn't we be able to look at `Statistics#isNumNullsSet` / `Statistics#getNumNulls` to know whether there are "official parquet nulls",  and then also need to check if there is overlap w/ deephaven representation of nulls in the `min / max` parquet range?